liblepton/scheme/unit-tests/t0402-config.scm

```diff
@@ -83,9 +83,9 @@
     (assert-equal a (config-load! a))
     (assert-true (config-loaded? a))
     (chmod *testdirAconf* #o000) ;; Make conf unreadable
-    (assert-thrown 'system-error (config-load! a)))
+    (assert-thrown 'system-error (config-load! a #t)))
```

Comment:
FWIW, here and in other places I would use something like `'force` or `'force-load` instead of just `#t` to improve readability.

Reply:
It's a good idea. Changed to `force-load` key parameter.

Reply:
He-he, I've meant just using `(config-load! a 'force-load)`. The symbol `'force-load` is always true. But fair enough, your new code is OK with me.